addons/packages/packetfence.spec

```diff
@@ -649,6 +649,11 @@ if [ ! -f /usr/local/pf/conf/pf_omapi_key ]; then
     /usr/bin/openssl rand -base64 -out /usr/local/pf/conf/pf_omapi_key 32
 fi
 
+# Create server local RADIUS secret
+if [ ! -f /usr/local/pf/conf/local_secret ]; then
+    date +%s | sha256sum | base64 | head -c 32 > /usr/local/pf/conf/local_secret
```

Comment:
Question is, should we really ?

pf_omapi_key doesn't do it this way so if I would change this one, pf_omapi_key needs to be changed as well to be consistent.

I personally don't see any problem having this using the default rights even though it is a key/password.

pf.conf + switches.conf contain that same type of info in clear text and they use the default rights.

Just playing the devils advocate so as even if you only have a semi-valid argument I'll change it :)